code/datums/ert.dm

```diff
@@ -49,7 +49,7 @@
 	rename_team = "Deathsquad"
 	code = "Delta"
 	mission = "Leave no witnesses."
-	polldesc = "an elite Nanotrasen Strike Team"
+	polldesc = "an overwhelmingly powerful and mysterious Strike Team"
```

Comment:
```suggestion
	polldesc = "a Death Commando Team"
```
tone it back girl this is supposed to be the default for what the ghosts are seeing